packages/ui/src/lib/tooltip/Tooltip.svelte

```diff
@@ -50,22 +50,47 @@
 </style>
 
 <script lang="ts">
+import type { Snippet } from 'svelte';
+
 import { tooltipHidden } from './tooltip-store';
 
-export let tip: string | undefined = undefined;
-export let top = false;
-export let topLeft = false;
-export let topRight = false;
-export let right = false;
-export let bottom = false;
-export let bottomLeft = false;
-export let bottomRight = false;
-export let left = false;
+interface Props {
+  tip?: string | Snippet;
+  top?: boolean;
+  topLeft?: boolean;
+  topRight?: boolean;
+  right?: boolean;
+  bottom?: boolean;
+  bottomLeft?: boolean;
+  bottomRight?: boolean;
+  left?: boolean;
+  class?: string;
+  'aria-label'?: string;
+  children?: Snippet;
+}
+let {
+  tip = undefined,
+  top = false,
+  topLeft = false,
+  topRight = false,
+  right = false,
+  bottom = false,
+  bottomLeft = false,
+  bottomRight = false,
+  left = false,
+  class: className = '',
+  'aria-label': ariaLabel,
+  children = undefined,
+}: Props = $props();
+
+function isSnippet(obj: string | Snippet): obj is Snippet {
+  return obj.length === 1;
+}
```

Comment:
This is a blocker for me, this is not officially supported and therefore could break in a near future - we should not allow to send anything else that a Snippet as children.